code/game/objects/items/trash.dm

```diff
@@ -76,9 +76,11 @@
 	icon = 'icons/obj/mining.dmi'
 	icon_state = "slag"
 	desc = "Someone's gotten on the naughty list."
-	grind_results = list("carbon" = 20)
+	grind_results = list("charcoal" = 20)
```

Comment:
Charcoal is made from wood. Coal is made from carbon. They are two different substances.